.eslintrc.js

```diff
@@ -1,196 +1,18 @@
+const { overrides } = require('@netlify/eslint-config-node')
+
 module.exports = {
-  parser: 'babel-eslint',
-  plugins: ['prettier', 'markdown', 'html'],
-  extends: [
-    'eslint:recommended',
-    'standard',
-    'prettier',
-    'prettier/standard',
-    'plugin:unicorn/recommended',
-    'plugin:eslint-comments/recommended',
-    'plugin:ava/recommended',
-    'plugin:react/recommended',
-    'prettier/react',
-    'plugin:you-dont-need-lodash-underscore/all',
-  ],
-  reportUnusedDisableDirectives: true,
+  extends: '@netlify/eslint-config-node',
   rules: {
-    'require-await': 2,
```

Comment:
All those rules and ESLint configuration are now sourced from `@netlify/eslint-config-node`